docs/source/modules/aws_s3_access_tutorial.rst

```diff
@@ -0,0 +1,124 @@
+.. _aws_s3_access_tutorial:
+
+**********************
+AWS S3 Access Tutorial
+**********************
+
+In order to make data assemblies accessible for Brain-Score model evaluations, they must be be uploaded. 
+You can self-host your data (e.g. on S3/OSF), or contact us to host your data on S3. You can also choose to keep your data private 
+such that models can be scored, but the data cannot be accessed.
+This tutorial walks through the process of uploading and downloading files to S3, and builds off of the information found in
+`the docs of BrainIO <https://github.com/brain-score/brainio/blob/main/docs/aws_access.md>`_.
+
+
+1. Configuration (see `aws_access <https://github.com/brain-score/brainio/blob/main/docs/aws_access.md>`_)
+==========================
+
+Although AWS credentials are unnecessary when downloading public files from S3 (just ask an admin for the information associated with the desired
+data assembly), AWS credentials are required when accessing private files or uploading files to S3. Ask an admin to create a user for you 
+and they will send you the credentials  necessary to configure your AWS access.
+
+After you have received your credentials, you can configure your access in two ways:
+
+
+Option 1: configuration via command line
+----------------------------------------
+Use `awscli` to configure your AWS access.
+
+.. code-block:: console
+
+    pip install awscli
+    aws configure
+
+(use region `us-east-1`, output-format `json`)
+
+
+Option 2: configuration via file
+----------------------------------------
+
+You can also create the configuration files yourself.
+
+Create your credentials file at `~/.aws/credentials`:
+
+.. code-block:: console
+
+    [default]
+    aws_access_key_id = YOUR_ACCESS_KEY_ID
+    aws_secret_access_key = YOUR_SECRET_ACCESS_KEY
+
+To configure region and output format, create a config file at `~/.aws/config`:
+
+.. code-block:: console
+
+    [default]
+    region = us-east-1
+    output-format = json
```

Comment:
Can we somehow point to https://github.com/brain-score/brainio/blob/main/docs/aws_access.md instead? Or integrate the text? I would like to avoid the redundancy.